sdks/python/apache_beam/coders/coders.py

```diff
@@ -650,6 +650,25 @@ def __hash__(self):
 Coder.register_structured_urn(common_urns.coders.VARINT.urn, VarIntCoder)
 
 
+class VarInt32Coder(FastCoder):
```

Comment:
can we add this to CHANGES.md?

Reply:
Thanks, will do as a follow up (otherwise need to rebase this PR onto latest master to be able to add to 2.65.0 section)

Reply:
opened #34408